commissions/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+
+from commissions import views
+
+urlpatterns = [
+    url(r'^$', views.ListCommissionsView.as_view(), name='list-commissions'),
```

Comment:
These urls should as indicated in the PR-comment lie on the `/api` url.

Reply:
Agree